examples/aws/storage-fsx/sample/fsx-pvc.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: fsx-claim
+  namespace: kubeflow-user-example-com
+spec:
+  accessModes:
+    - ReadWriteMany
+  storageClassName: ""
```

Comment:
Is this suppose to be empty ? unlike efs ?

Reply:
yes, it uses the default storageclass in this case.